build_aarch64_wheel.py

```diff
@@ -36,7 +38,7 @@ def ec2_instances_by_id(instance_id):
     return rc[0] if len(rc) > 0 else None
 
 
-def start_instance(ami=ubuntu18_04_ami, instance_type='t4g.2xlarge', key_name='nshulga-key'):
+def start_instance(ami=ubuntu18_04_ami, instance_type='t4g.2xlarge', key_name=None):
```

Comment:
Hmm, defaulting `key_name` to `None` seems wrong...

Reply:
we can remove the default and move the argument to the front?